src/locales/en.json

```diff
@@ -78,7 +78,7 @@
     "group": "Group selection",
     "ungroup": "Ungroup selection",
     "collaborators": "Collaborators",
-    "showGrid": "Show grid",
+    "showGrid": "Show grid (Cmd/Ctrl: No Snap)",
```

Comment:
this should instead go to the toolbar hint

Reply:
Do you mean add it here on condition gridSize !== null? What happens when multiple hints are to be displayed?

https://github.com/excalidraw/excalidraw/blob/6abf1942814ec002f916b8a17c52dd124531bc42/src/components/HintViewer.tsx#L21-L111